File: aula4/java/Exercicio3.java
import java.util.Scanner;

public class Exercicio3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] A = new int[15];
        int[] B = new int[15];

        for(int i = 0; i < 15; i++){
            System.out.print("Digite um número: ");
            A[i] = sc.nextInt();
        }

        for(int i = 0; i < 15; i++){
            B[i] = A[i] * A[i];
        }

        for(int i = 0; i < 15; i++){
            System.out.println("A: " + A[i] + " B: " + B[i]);
        }

        sc.close();
    }
}

File: aula4/cpp/exercicio3.cpp
#include <iostream>
using namespace std;

int main() {
    int A[15], B[15];

    for(int i = 0; i < 15; i++){
        cout << "Digite um número: ";
        cin >> A[i];
    }

    for(int i = 0; i < 15; i++){
        B[i] = A[i] * A[i];
    }

    for(int i = 0; i < 15; i++){
        cout << "A: " << A[i] << " B: " << B[i] << endl;
    }

    return 0;
}
